Flank/src/main/java/com/walmart/otto/reporter/TimeReporter.java

```diff
@@ -28,4 +28,8 @@ public static String getEndTime() {
         Date date = new Date();
         return simpleDateFormat.format(date);
     }
+
+    public static List<Integer> getExecutionTimes(){
+        return executionTimes;
+    }
 }
```

Comment:
is there a linting package in java? I work with javascript mostly, so not very sure. Hate to fix these manually.